src/main/java/io/neonbee/NeonBee.java

```diff
@@ -339,8 +343,12 @@ Future<Void> registerHealthChecks() {
         if (Optional.ofNullable(config.getHealthConfig()).map(HealthConfig::isEnabled).orElse(true)) {
             healthChecks.add(healthRegistry.register(new MemoryHealthCheck(this)));
 
-            if (options.isClustered()) {
-                healthChecks.add(healthRegistry.register(new HazelcastClusterHealthCheck(this, clusterManager)));
+            if (options.isClustered() && vertx instanceof VertxInternal) {
```

Comment:
What do you mean with "trust"? We can also rely on *options.isClustered()*, otherwise we have a ctricial bug in NeonBee.

I'd prefer *options.isClustered()* because of two reasons:
- before we also relied on *options.isClustered()*
- it is much faster then instance of

Reply:
Changed to your suggestion (as discussed)